src/main/resources/sql/create.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE IF NOT EXISTS "shop" (
+  "id"      SERIAL PRIMARY KEY,
+  "name"    VARCHAR(255)        NOT NULL,
+  "address" VARCHAR(255) UNIQUE NOT NULL
```

Comment:
1. What if two stores are located at one address?
2. Maybe make the "name" field is unique?